pyuavcan/transport/serial/_frame.py

```diff
@@ -7,9 +7,9 @@
 from __future__ import annotations
 import typing
 import struct
-import itertools
 import dataclasses
 import pyuavcan
+from cobs import cobs
```

Comment:
This is to suppress the MyPy warning about the type hints being unavailable.

```suggestion
from cobs import cobs    # type: ignore
```

Reply:
yes, just committed this fix